pytorch_lightning/metrics/utils.py

```diff
@@ -36,8 +37,8 @@ def _flatten(x):
 
 
 def to_onehot(
-        tensor: torch.Tensor,
-        num_classes: int,
+    tensor: torch.Tensor,
```

Comment:
```suggestion
    data: torch.Tensor,
```
can we name it better then tensor so it is not confusing with PT data type, well suggested `data` is also not a good one :]

Reply:
Changed it to `labels_tensor`